src/main/java/me/glasscrab/plentiful_harvest/Listeners/AnimalFeedEvent.java

```diff
@@ -20,29 +25,34 @@ public void feedAnimal(PlayerInteractEntityEvent e) {
         if(e.getHand().equals(EquipmentSlot.OFF_HAND)) return;
 
         ItemStack hand = e.getPlayer().getInventory().getItemInMainHand();
+        var miniMessage = MiniMessage.miniMessage();
+        Audience audience = PlentifulHarvest.INSTANCE.audiences.player(e.getPlayer());
 
         if(isSuperCrop(hand)) {
             e.setCancelled(true);
 
             if(hand.getType().equals(Material.POTATO) && e.getRightClicked() instanceof LivingEntity && !e.getPlayer().isSneaking()){
                 ((LivingEntity) e.getRightClicked()).setHealth(((LivingEntity) e.getRightClicked()).getAttribute(Attribute.GENERIC_MAX_HEALTH).getValue());
-                e.getPlayer().getWorld().spawnParticle(Particle.VILLAGER_HAPPY, ((LivingEntity) e.getRightClicked()).getEyeLocation(),15,.3,.3,.3);
+                e.getPlayer().getWorld().spawnParticle(Particle.HAPPY_VILLAGER, ((LivingEntity) e.getRightClicked()).getEyeLocation(),15,.3,.3,.3);
                 e.getPlayer().playSound(e.getPlayer(), Sound.BLOCK_AMETHYST_BLOCK_RESONATE, SoundCategory.MASTER, 1,1);
-                e.getPlayer().spigot().sendMessage(ChatMessageType.ACTION_BAR, TextComponent.fromLegacyText(ChatColor.GREEN+""+e.getRightClicked().getType()+" has been healed!"));
+                Component parsedText = miniMessage.deserialize("<green>" + e.getRightClicked().getType() + " has been healed!</green>");
+                audience.sendActionBar(parsedText);
```

Comment:
Same as I said in the command class, just use e#getPlayer since the console won't be breaking crops